File: app/src/main/java/com/dabinu/apps/chesstimer/MainActivity.java
package com.dabinu.apps.chesstimer;

import android.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        FragmentTransaction fragmentTransaction = getFragmentManager().beginTransaction();
        fragmentTransaction.replace(R.id.container, new MenuFragment());
        fragmentTransaction.commit();
    }


    @Override
    public void onBackPressed() {
        FragmentTransaction fragmentTransaction = getFragmentManager().beginTransaction();
        fragmentTransaction.replace(R.id.container, new MenuFragment());
        fragmentTransaction.commit();
    }
}

File: app/src/main/java/com/dabinu/apps/chesstimer/MainActivity.kt
package com.dabinu.apps.chesstimer


import android.support.v7.app.AppCompatActivity
import android.os.Bundle

import com.dabinu.apps.chesstimer.fragments.MenuFragment

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val fragmentTransaction = fragmentManager.beginTransaction()
        fragmentTransaction.replace(R.id.container, MenuFragment())
        fragmentTransaction.commit()
    }


    override fun onBackPressed() {
        val fragmentTransaction = fragmentManager.beginTransaction()
        fragmentTransaction.replace(R.id.container, MenuFragment())
        fragmentTransaction.commit()
    }
}